crates/meilisearch/tests/index/get_index.rs

```diff
@@ -26,13 +26,13 @@ async fn create_and_get_index() {
 
 #[actix_rt::test]
 async fn error_get_unexisting_index() {
-    let server = Server::new().await;
-    let index = server.index("test");
+    let server = Server::new_shared();
+    let index = server.unique_index();
```

Comment:
I believe you can directly use the special [`shared_does_not_exists_index`](https://github.com/meilisearch/meilisearch/blob/8f006eeaf3c001a0fe72a43772bfb871e588debf/crates/meilisearch/tests/common/mod.rs#L163C14-L169) here 

```suggestion
    let index = shared_does_not_exists_index();
```